transformation/src/test/resources/log4j2.properties

```diff
@@ -1,4 +1,4 @@
-status = error
```

Comment:
Rather than modify all of these seperately can we have a shared configuration, or at least a base version of this file?

Reply:
Do you have a recommendation for how to package that into all of our builds and IDE settings?

Reply:
I tried playing around with this through gradle settings to specify multiple files in the log4j.configurationFile and wasn't able to get it to work. It seemed like it wasn't able to access files outside the resources directory. We can experiment in the future, but i'd rather not block this PR on it which allows us to test the tracing side effects